files/docker/bin/dockerd-wrapper

```diff
@@ -60,11 +60,18 @@ workaround_lp1606510
 
 workaround_apparmor_profile_reload
 
+if [ $(snapctl get docker.bridge) = "disable" ]; then
```

Comment:
We have other examples where we use double-brackets (i.e. [[  ]] ). Not sure if there's really a difference.  ping @CristianPrundeanuARM ?